crossplane-argocd-gitops/envs/dev/values.yaml

```diff
@@ -0,0 +1,18 @@
+spec:
+  destination:
+    server: https://kubernetes.default.svc
+  source:
+    repoURL: https://github.com/aws-samples/eks-blueprints-workloads
+    targetRevision: main
+clusterA:
+  clusterName: amd-1-29-blueprint
+clusterB:
+  clusterName: arm-1-29-blueprint  
+common:
+  provider-config-aws-name: common-provider-config-aws
+  eks-connector-role-name: eks-connector-role
+  account_id: <enter_accont_id_here>
```

Comment:
Lets not hard code this in a GitOps based approach, then is not fully automated. lets pass it via argo to GitOps. Same with region var.

Reply:
yes done